src/server/routes.ts

```diff
@@ -17,6 +17,20 @@ router.post('/plants', async (req, res, next) => {
   }
 });
 
+router.get('/plants', async (_req, res, next) => {
+  try {
+    const plants = await findPlants();
+    res.json(plants);
```

Comment:
```suggestion
```
You are sending the plants before checking if they even exist

Reply:
res.status(200).json(plants); Did you mean that?